client/wordpress-com.js

```diff
@@ -157,7 +150,7 @@ const sections = [
 		name: 'themes',
 		paths: [ '/design' ],
 		module: 'my-sites/themes',
-		enableLoggedOut: config.isEnabled( 'manage/themes/logged-out' ),
+		enableLoggedOut: true,
```

Comment:
Since `manage/themes/logged-out` is true in every environment configuration I think that modifying this should be harmless.  Is this true?

cc @seear